src/main/scala/com/gu/subscriptions/cas/directives/HealthCheckDirective.scala

```diff
@@ -1,21 +1,16 @@
 package com.gu.subscriptions.cas.directives
 
-import com.gu.subscriptions.cas.config.Configuration.appConfig
+import com.gu.subscriptions.cas.config.Configuration.nullSettings
 import spray.http.MediaTypes._
 import spray.routing.{HttpService, Route}
 
 trait HealthCheckDirective {this: HttpService =>
-
-  val checkConfig = List(
-    "proxy"
-  ).filter(appConfig.getString(_) == null)
-
```

Comment:
I would keep this logic here rather than in Config. It's only used here and not really a config property.